ext/tags.go

```diff
@@ -22,6 +22,9 @@ var (
 
 	// PeerPort records port number of the peer
 	PeerPort = uint16Tag("peer.port")
+
+	// SamplingPriority determines the priority of sampling this Span.
+	SamplingPriority = uint32Tag("sampling.priority")
```

Comment:
that's how it sort of was in the initial version of this file, then it got changed to current form that was deemed more streamlines / Go-like. I do like the ability to hide the string value to avoid downcasting abuse.